code/modules/clothing/suits/armor_suits.dm

```diff
@@ -628,7 +628,7 @@
 
 		for(var/am in thrown_atoms)
 			var/atom/movable/AM = am
-			if(AM == owner || AM.anchored)
+			if(AM == owner || AM.anchored || isobserver(AM))
```

Comment:
```suggestion
			if(AM == owner || AM.anchored || (ismob(AM) && !isliving(AM)))
```
Need this else camera mobs can get thrown too

Reply:
Done, ty for your time!